index.html

```diff
@@ -0,0 +1,394 @@
+                        <li class="side-calendar__day">23</li>
+                        <li class="side-calendar__day">24</li>
+                        <li class="side-calendar__day">25</li>
+                        <li class="side-calendar__day">26</li>
+                        <li class="side-calendar__day">27</li>
+                        <li class="side-calendar__day">28</li>
+                        <li class="side-calendar__day">29</li>
+                    </ul>
+
+                    <ul class="side-calendar__list">
+                        <li class="side-calendar__day">30</li>
+                        <li class="side-calendar__day">31</li>
+                        <li class="side-calendar__day not-curr-month">1</li>
+                        <li class="side-calendar__day not-curr-month">2</li>
+                        <li class="side-calendar__day not-curr-month">3</li>
+                        <li class="side-calendar__day not-curr-month">4</li>
+                        <li class="side-calendar__day not-curr-month">5</li>
+                    </ul>
+                </div>
+            </section>
+        </aside>
+
+        <section class="lg-calendar-wrapper">
+
+            <span class="lg-calendar__gmt">gmt+03</span>
+
+            <ul class="lg-calendar__column">
+                <li>
+                    <span>sun</span>
+                    <span>9</span>
+                </li>
+                <li>
+                    <span>mon</span>
+                    <span>10</span>
+                </li>
+                <li>
+                    <span>tue</span>
+                    <span>11</span>
+                </li>
+                <li>
+                    <span>wed</span>
+                    <span>12</span>
+                </li>
+                <li>
+                    <span>thu</span>
+                    <span>13</span>
+                </li>
+                <li>
+                    <span>fri</span>
+                    <span>14</span>
+                </li>
+                <li>
+                    <span>sat</span>
+                    <span>15</span>
+                </li>
+            </ul>
+
+            <ul class="lg-calendar__hour-wrapper">
+                <li>1 am</li>
+                <li>2 am</li>
+                <li>3 am</li>
+                <li>4 am</li>
+                <li>5 am</li>
+                <li>6 am</li>
+                <li>7 am</li>
+                <li>8 am</li>
+                <li>9 am</li>
+                <li>10 am</li>
+                <li>11 am</li>
+                <li>12 pm</li>
+                <li>1 pm</li>
+                <li>2 pm</li>
+                <li>3 pm</li>
+                <li>4 pm</li>
+                <li>5 pm</li>
+                <li>6 pm</li>
+                <li>7 pm</li>
+                <li>8 pm</li>
+                <li>9 pm</li>
+                <li>10 pm</li>
+                <li>11 pm</li>
+            </ul>
+
+            <ul class="lg-calendar__row">
```

Comment:
`class="lg-calendar__row"` it's kind of not a row if it holds all the cells. Maybe `lg-calendar__table` or `lg-calendar__cells`?